clang_7/Dockerfile

```diff
@@ -27,7 +27,7 @@ RUN dpkg --add-architecture i386 \
        nasm=2.13.03-2 \
        dh-autoreconf=19 \
        libffi-dev=3.2.1-8 \
-       libssl-dev=1* \
+       libssl-dev=1.1.1-1ubuntu2 \
```

Comment:
I admin I relaxed some versions checks because it is a pain in the ass, passed some time the build fails because that package is conflicting or not available anymore. Which the best practices? What is the best way to know the correct version to pin?

Reply:
Maybe we should reduce the pinning to major or minor version, but removing the patch and build number.

Reply:
Our docker linter strongly recommends a complete version tag, but how many times have we had any problems related to a bad package version? 2 years running docker + conan and I saw once.

I see more effort trying to keep the same environment versioned than avoiding bugs from from apt. It's not wasted time, but I vote for relaxed version, using major only. Otherwise, we don't have power to keep the same version when ubuntu or any apt source update their packages.